frontend/src/pages/ErrorsV2/ErrorInstance/ErrorInstance.tsx

```diff
@@ -278,9 +257,6 @@ const ErrorInstance: React.FC<Props> = ({ errorGroup }) => {
 		)
 	}
 
-	const errorObject =
-		errorInstance.error_object as ErrorInstanceType['error_object']
```

Comment:
This casting wasn't actually correct. We don't have an `ErrorObject` but rather a `GetErrorObjectQuery['error_object']`. 

Using the graphql fragment makes the typing correct.